test/ApplicationInsightsTypes/Generated/bonddefaultcodegen.in

```diff
@@ -0,0 +1,19 @@
+..\..\..\Schema\PublicSchema\AvailabilityData.bond
```

Comment:
add this file in `.gitignore` and remove from PR

Reply:
done